paychmgr/simple.go

```diff
@@ -291,10 +295,65 @@ func (ca *channelAccessor) msgWaitComplete(mcid cid.Cid, err error) {
 	// The queue may have been waiting for msg completion to proceed, so
 	// process the next queue item
 	if len(ca.fundsReqQueue) > 0 {
-		go ca.processQueue()
+		go ca.processQueue() // nolint: errcheck
 	}
 }
 
+func (ca *channelAccessor) currentAvailableFunds(queuedAmt types.BigInt) (*api.ChannelAvailableFunds, error) {
+	channelInfo, err := ca.store.OutboundActiveByFromTo(ca.from, ca.to)
+	if err != nil && err != ErrChannelNotTracked {
+		return nil, err
+	}
+
+	if channelInfo == nil {
```

Comment:
In what state is channelInfo == nil here? Is this not something that should return an error?

Reply:
channelInfo == nil when we get ErrChannelNotTracked (see line 304 above).

That means that we don't have any information about a channel with the given from / to pair. I think we don't want to return an error here, because a legitimate use case is to use `PaychAvailableFunds` to determine whether a channel exists between `from` and `to`

I will add a comment explaining this.